src/main/java/core/basesyntax/exeptions/ReportsListEmptyException.java

```
@@ -0,0 +1,6 @@
+package core.basesyntax.exeptions;
+public class ReportsListEmptyException extends RuntimeException {
+    public ReportsListEmptyException(String message) {
+        super(message);
```

Missing cause-preserving constructors. The project checklist requires rethrowing exceptions with the original cause: "Don't ignore exceptions in try-catch blocks ... Let's rethrow a RuntimeException with an **informative** message and exception object." Add constructors `ReportsListEmptyException(String message, Throwable cause)` and `ReportsListEmptyException(Throwable cause)` so callers can preserve underlying exceptions (consistent with other exceptions such as IncorrectFormatOfDataException).